CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 
 - tbd
 
+### Added
+- New detector `FindHidingSubClass` for bug type `HSBC_HIDING_SUB_CLASS`. This bug is reported whenever a subclass method hides the static method of super class. (See [SEI CERT MET07-J] (https://wiki.sei.cmu.edu/confluence/display/java/MET07-J.+Never+declare+a+class+method+that+hides+a+method+declared+in+a+superclass+or+superinterface)).
```

Comment:
I think you forgot to edit the changelog after changing the name of the detector, thank you for all the changes!

Reply:
Yeah, it just slipped out of my mind. I have fixed it now. Thank you for your prompt review :))